news/2018-11-21-new-featured-artist-culprate.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: "New Featured Artist: Culprate"
+date: 2018-11-21 10:00:00 +0000
+---
+
+We proudly welcome an extremely talented producer hailing from the United Kingdom to the osu! Featured Artist lineup — Culprate!
+
+[![](https://assets.ppy.sh/artists/29/header.jpg)](https://osu.ppy.sh/beatmaps/artists/29)
+
+Something of a standout among the UK's underground dubstep scene, **Culprate** is a tremendously capable music producer with some incredible musical clout. Renowned for his incredible production quality and captivatingly complex tracks, we pretty much fell in love in his sound when we first heard it.
+
+One of **Culprate**'s most recent projects features a massive multi-work collaboration drawn from his Twitch community, of which one of our Featured Artists (namely [Dictate](https://osu.ppy.sh/beatmaps/artists/21)) was a part of. Aptly titled as the *Unity Project*, we are proud to announce that a selection from this body of work and from **Culprate**'s cult classic album *Deliverance* are now available.
+
+The Mappers' Guild has been hard at work over the past few months drawing up some incredible maps to help inspire both players and mappers alike with **Culprate**'s sound, check them out — completing all six will nab you the 4th special Mapper's Guild medal! Collecting them all will make you the envy of your friends. Probably.
+
+*Clicking on the image below will start a download for the pack of all six maps mentioned!*
+
+[![](/wiki/shared/news/2018-11-21-new-featured-artist-culprate/listing.jpg)](https://assets.ppy.sh/beatmap-packs/MappersGuildCulprate.zip)
+
+Here's a sample of what's we've got in store for you to map to:
+
+### Acid Rain
+
+*This track was featured as the pick for Aspire 2018.*
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/29/previews/624.mp3" type="audio/mpeg">
+</audio>
+
+### Relucent (featuring ZES)
+
+*A special bonus track on the end of the digital edition of Deliverance, Relucent is a celebration of all the acoustic-electronic fusion displayed throughout the album.*
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/29/previews/631.mp3" type="audio/mpeg">
+</audio>
+
+### Impulse (featuring Au5)
+
+*From the second part of the Unity Project albums is perhaps one of the most iconic osu! tracks in recent years, thanks to [handsome's award winning map](https://osu.ppy.sh/beatmapsets/705788#osu/1492654).*
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/29/previews/625.mp3" type="audio/mpeg">
+</audio>
+
+### Pencilina (featuring Dictate)
+
+*From part two of the Unity Project is a collaboration track with our own Dictate, merging their two distinct sounds to produce something very, very special.*
```

Comment:
This sentence is basically the same as the previous but changing "second part" to "part two", maybe rewrite?

Reply:
From part two..., this collaboration track with our own Dictate merges their two distinct sounds to produce something very, very special.